src/main/java/org/quiltmc/loader/impl/util/StringUtil.java

```diff
@@ -29,4 +32,8 @@ private StringUtil() {
 			return new String[] { defaultNamespace, s };
 		}
 	}
+
+	public static String either(@Nullable String candidate, @NotNull String orElse) {
+		return (candidate == null || candidate.isEmpty()) ? orElse : candidate;
```

Comment:
A tenary operator or an if statement should suffice here. This util method doesnt see enough use imo for it.